hazelcast/src/test/java/com/hazelcast/map/EntryCostEstimatorTest.java

```diff
@@ -39,22 +39,29 @@
         extends HazelcastTestSupport {
 
     protected TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory(2);
-    // the JVM-independent portion of the cost of Integer key + Long value record is 104 bytes
-    // (without taking into account 8 references to key, record and value objects)
-    private static final int JVM_INDEPENDENT_ENTRY_COST_IN_BYTES = 100;
-    // JVM-dependent total cost of entry
-    private static final int ENTRY_COST_IN_BYTES = JVM_INDEPENDENT_ENTRY_COST_IN_BYTES + 8 * REFERENCE_COST_IN_BYTES;
+
+    public static final int ENTRY_COST_IN_BYTES = getExpectedCostInBytes();
+
+    private static int getExpectedCostInBytes() {
+        if (JVMUtil.is32bitJVM() && JVMUtil.isCompressedOops()) {
+            return 140;
```

Comment:
Maybe keeping the math with the comments we had before would be better.

Reply:
The math is highly dependent to internals and this makes it hard to express after refactorings/changes. I found the values here by logging entry costs, this seemed easy to me.